src/browser/modules/Sidebar/GuidesDrawer.tsx

```diff
@@ -51,21 +51,26 @@ function GuidesDrawer({
   gotoSlide
 }: GuidesDrawerProps): JSX.Element {
   const scrollRef = useRef<HTMLDivElement>(null)
-  const title = guide.title.startsWith('http')
-    ? guide.title
-    : guide.title.charAt(0).toUpperCase() + guide.title.slice(1)
 
   return (
     <StyledGuidesDrawer id="guide-drawer" ref={scrollRef}>
       <StyledGuidesDrawerHeader onClick={backToAllGuides}>
-        {guide.title !== defaultGuide.title && (
+        {!isDefaultGuide(guide) && (
           <BackIconContainer>
             <BackIcon width={16} />
           </BackIconContainer>
         )}
         Neo4j Browser Guides{' '}
       </StyledGuidesDrawerHeader>
-      <GuideTitle title={title}>{title}</GuideTitle>
+      <div
+        style={{
```

Comment:
Can these styles be moved to `styled.tsx` or what not?

Reply:
Yeah, I forgot it! Moving it now..